src/include/error.h

```diff
@@ -123,82 +123,124 @@
     __wt_panic(session, EINVAL, "%s: 0x%" PRIxMAX, \
```

Comment:
This diff is a bit confusing. I'd recommend viewing in side-by-side view rather than inline.